### PR TITLE
Mtd 1964 what can i do api endpoint

### PR DESCRIPTION
**Pull request by zuzsso**

### CODE REVIEW

File: src/JwtGuard/JwtTokenAuthenticator.php [hunk 1]
```diff
@@ -20,6 +20,7 @@ class JwtTokenAuthenticator implements RequestAuthoriser, IdentityProvider
     const IS_ADMIN = 'isAdmin';
     const SEGMENTS = 'segments';
     const BEARER = 'Bearer ';
+    const DB_ID = 'dbId';
```
Comment (peter-horvath) [inline src/JwtGuard/JwtTokenAuthenticator.php]: I think it was a concern at the time this was developed, whether we should use the DB ID or the passport group as the unique identifier for companies. I think for some reason whoever developed this, choose the passport group and as far as I can remember it was a conscious decision  to expose only one unique ID.
I guess if the service requiring this data uses DB ID then we can't do too much and have to do this now...

**Review by peter-horvath — APPROVED** [review]

File: src/JwtGuard/JwtTokenAuthenticator.php (hunk 1 above)
Comment (zuzsso) [reply src/JwtGuard/JwtTokenAuthenticator.php]: We passing the `DbID` anyway in the JWT token, and that can be seen by anyone intercepting it. Perhaps that is not a concern anymore... besides, how a seemingly random subrogated ID could reveal any information about the user?

Comment (peter-horvath) [reply src/JwtGuard/JwtTokenAuthenticator.php]: @zuzsso ah I wasn't concerned about revealing anything to the user. My only problem was about exposing a technical debt to a new API, and keep using two unique identifiers for the same record (passport group and DB ID).
But yeah, never mind me, this ship has already gone. We're unable to stop spreading this bad design to the API at this point, it was already done earlier :D

Comment (zuzsso) [reply src/JwtGuard/JwtTokenAuthenticator.php]: Understand... Undo then?

Comment (peter-horvath) [reply src/JwtGuard/JwtTokenAuthenticator.php]: @zuzsso no, not at all. Like you said we were already passing the DB ID to the JWT token and I guess we're relying on it in external services, so this change doesn't make it worse.

**Pull request merged**